nan.h

```diff
@@ -1393,6 +1393,15 @@ class NanCallback {
     NanDisposePersistent(handle);
   }
 
+  friend bool operator==(NanCallback &cb1, NanCallback &cb2) {
+    if(cb1.IsEmpty() && cb2.IsEmpty()) return 1;
```

Comment:
Style issues: space after if and true instead of 1.  Using `friend` for a member method is superfluous.  There's also a logic error in the IsEmpty() checks, it should be:

```c++
if (cb1.IsEmpty() || cb2.IsEmpty()) {
  return cb1.IsEmpty() && cb2.IsEmpty();
}
return cb1.GetFunction() == cb2.GetFunction();
```

Or maybe just rewrite it as:

```c++
// More efficient.
Local<Value> a = NanNew(cb1.handle)->Get(kCallbackIndex);
Local<Value> b = NanNew(cb2.handle)->Get(kCallbackIndex);
return a->StrictEquals(b);
```